src/Components/UI/Button/Button.jsx

```diff
@@ -0,0 +1,15 @@
+const Button = (props) => {
+  return (
+    <div className={props.btnWrapper}>
```

Comment:
props here could be destructed as it is used more than once in functionality

Reply:
Noted